sandbox/apparmor/apparmor_test.go

```diff
@@ -41,6 +41,18 @@ type apparmorSuite struct{}
 
 var _ = Suite(&apparmorSuite{})
 
+func (*apparmorSuite) TestAppArmorFindAppArmorParser(c *C) {
```

Comment:
we also need the inverse unit test too, namely that if apparmor_parser exists in the internal tool path that `internal` is `true`.

Reply:
I agree - I am just not sure how to add a test which runs under this circumstance - ie. a snapd snap installed from this PR - can you advise how I should structure this?

Reply:
sure, here I was just suggesting to mock some things in the unit tests, let me know if you need help using the MockCommand and SetRootDir etc functions to mock this for the unit test, I'll respond about spread tests separately

Reply:
Ah ok I'll see what I can cook up - cheers.

Reply:
Ok I added something in 6686892320